trunk/src/modules/element_data.F

```diff
@@ -1183,7 +1186,9 @@ subroutine hexa_aux(Iv1,Iv2, Iface)
       integer,intent(out) :: Iface
 c
       integer,save,dimension(8,8) :: AUX
+c$omp threadprivate(AUX)
       logical,save                :: INITIALIZED=.FALSE.
+c$omp threadprivate(INITIALIZED)
```

Comment:
@jbadger95 were these changes to `element_data` meant to be part of the PR? I can't see they're used anywhere?

Reply:
Yes, they're needed the first time threaded `update_gdof` is called

Reply:
`update_gdof->hpvert->hexa->hexa_TraHex->hexa_aux`

Reply:
sounds good, thanks